src/dev-app/dev-app/dev-app-module.ts

```diff
@@ -32,6 +34,7 @@ import {DevAppLayout} from './dev-app-layout';
   ],
   declarations: [DevAppLayout, DevAppHome, DevApp404],
   exports: [DevAppLayout],
+  providers: [{provide: HAMMER_GESTURE_CONFIG, useClass: GestureConfig}]
```

Comment:
That is related to https://github.com/angular/material2/issues/4595#issuecomment-416641018. 

Optional: add a small comment so we know why this is necessary.